spec/sidekiq/ultimate/resurrector_spec.rb

```diff
@@ -57,7 +92,60 @@
 
         expect(resurrections).to contain_exactly(["queue2", 2])
         jobs_in_queue = Sidekiq.redis { |redis| redis.lrange("queue:queue2", 0, -1) }
-        expect(jobs_in_queue).to match_array(%w[sidekiq_job_hash1 sidekiq_job_hash2 sidekiq_job_hash3])
+        expect(jobs_in_queue).to contain_exactly(job_hash2, job_hash3, job_hash4)
+      end
+
+      it "increments the resurrection counter when enable_resurrection_counter is true" do
+        allow(Sidekiq::Ultimate::Configuration.instance).
+          to receive(:enable_resurrection_counter).and_return(-> { true })
+
+        resurrect!
+
+        counter1 = Sidekiq.redis { |r| r.get("ultimate:resurrector:counter:jid:2647c4fe13acc692326bd4c2") }
+        counter1_ttl = Sidekiq.redis { |r| r.ttl("ultimate:resurrector:counter:jid:2647c4fe13acc692326bd4c2") }
+        counter2 = Sidekiq.redis { |r| r.get("ultimate:resurrector:counter:jid:2647c4fe13acc692326bd4c3") }
+        counter2_ttl = Sidekiq.redis { |r| r.ttl("ultimate:resurrector:counter:jid:2647c4fe13acc692326bd4c3") }
+
+        expect(counter1).to eq("1")
+        expect(counter1_ttl).to be_within(5).of(86_400)
+        expect(counter2).to eq("1")
+        expect(counter2_ttl).to be_within(5).of(86_400)
+      end
+
+      it "does not increment the resurrection counter when enable_resurrection_counter is not set" do
```

Comment:
Probably not `enable_resurrection_counter is not set`, but `enable_resurrection_counter proc returns 'false'`

Reply:
https://github.com/sensortower/sidekiq-ultimate/pull/4/commits/888b899e03cd7717d1491ebe03cb5fb0d47d5eb5